.github/workflows/run_tests.yaml

```diff
@@ -6,7 +6,25 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - run: |
+          sudo apt-get update
+          sudo apt-get install -y conntrack
+      - uses: medyagh/setup-minikube@latest
+        with:
+          driver: docker
+          container-runtime: containerd
+          kubernetes-version: v1.21.9
```

Comment:
why not 1.23?

Reply:
potentially current prometheus-operator and grafana-operator not works on 1.22+, we need upgrade it first in separate PR it need research, for example grafana-operator 4.x version broke installation process compatible with 3.9.x